lib/core/logconf.c

```diff
@@ -63,6 +63,7 @@ enum {
     ELEMENT_TYPE_PROCESS_TIME,                  /* %{process-time}x */
     ELEMENT_TYPE_RESPONSE_TIME,                 /* %{response-total-time}x */
     ELEMENT_TYPE_DURATION,                      /* %{duration}x */
+    ELEMENT_TYPE_TOTAL,                         /* %{total}x */
```

Comment:
Same here. Both `%{duration}x` and `%{total-time}x` can map to the same enum element.